src/Endatix.Core/Integrations/Slack/ISlackClient.cs

```diff
@@ -1,10 +1,13 @@
+using Endatix.Core.Events;
+
 namespace Endatix.Core.Integrations.Slack;
 
 public interface ISlackClient {
     /// <summary>
     /// Posts a message to a Slack channel through the Endatix Bot Slack app
     /// </summary>
-    /// <param name="message">The body of the message</param>
+    /// <param name="notification">The notification object passed by the event publisher</param>
+    /// /// <param name="cancellationToken">The event's cancellation token</param>
     /// <returns>Task</returns>
-    private Task PostMessageAsync(string message);
+    public Task Handle(SubmissionCompletedEvent notification, CancellationToken cancellationToken);
```

Comment:
A couple of things to mention, no need to be changed now, just to have in mind:
- Interface members don't need access modifiers - they are considered public by default. I even expected that explicitly using `public` will fail the build but it doesn't have problem.
- This method returns Task so it's async method. A good practice is to end their names with `Async`, like e.g. `HandleAsync`. We don't follow this everywhere in our code base but it's good to try following it.

Reply:
I agree. As I mentioned, we're not quite consistent on the naming of async methods. It would be good to sync about this in the team. Thanks.